### DISCUSSION TITLE
Should we remove CHV_APP_PATH_LIB_VENDOR in loader.php

### DISCUSSION THREAD

**Discussion by DawnAce** [Chevereto/Chevereto-Free#146] [General]
The latest version doesn't have /vendor folder under app. If we keep the code associated with CHV_APP_PATH_LIB_VENDOR in loader.php, there will be some problems when installing.
![image](https://user-images.githubusercontent.com/19185252/148666609-af7f4e76-a911-48bf-936e-db21e184be08.png)

**Comment by rodber** [comment]
https://github.com/rodber/chevereto-free/blob/11718635a4b223dc3ebec94dd60d9dc72630fe77/composer.json#L31
